docs/source/common/trainer.rst

```diff
@@ -579,6 +597,24 @@ based on the accelerator type (``"cpu", "gpu", "tpu", "ipu", "auto"``).
     # Training with IPU Accelerator using 4 ipus
     trainer = Trainer(devices="auto", accelerator="ipu")
 
+.. note::
+    If the `devices` flag is not defined, it will assume `devices`` to be `"auto"` and fetch the `auto_device_count`
```

Comment:
I think this is already mentioned in L-584 as a tip .

Reply:
I think it makes sense to mention it for both `accelerator` and `devices` section

Reply:
Merge both in what ways?

Reply:
I think it makes sense to keep it separate.

Reply:
also back-ticks needs to be fixed :)

Reply:
Fixed!